README.md

```diff
@@ -71,6 +71,20 @@ app.use(
 )
 ```
 
+#### Note on GET_LIST hooks
```

Comment:
I'm not sure it deserves a not. Updating the API line 56 seems enough. What do you think about it?